.syncpackrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
```

Comment:
Let's include types here like in this snippet:

https://jamiemason.github.io/syncpack/config-file#typescript-intellisense

(Applies to all config files that have types provided)

Reply:
Ah yes, thanks, done.